### ISSUE TITLE
Need more output from cmake

### ISSUE THREAD

**Issue by rljacob** [E3SM-Project/E3SM#3758]
When running ./case.build interactively, this is some of what the user sees output to the screen:
```
Building pio with output to file /compyfs/jaco316/e3sm_scratch/Ftest/bld/pio.bldlog.200804-185823
   Calling /qfs/people/jaco316/E3SM3/cime/src/build_scripts/buildlib.pio
Building csm_share with output to file /compyfs/jaco316/e3sm_scratch/Ftest/bld/csm_share.bldlog.200804-185823
   Calling /qfs/people/jaco316/E3SM3/cime/src/build_scripts/buildlib.csm_share
Component csm_share build complete with 20 warnings

Time spent not building: 2.689481 sec
Time spent building: 581.197731 sec
MODEL BUILD HAS FINISHED SUCCESSFULL
```
There is nothing output to the screen for several minutes after the "Component csm_share..." message because at that point cmake takes over and builds the rest of the model including the time-consuming full models.

It would be good if the build system output a message listing the components cmake was building and where to find the log file.

**Comment by worleyph** [comment]
@rljacob , I asked this question of @jgfouca when CMake was first introduced - there are no separate build logs for components any more (anything built with CMake). Everything is dumped into e3sm.log, and there is also no separation between (i.e., no timing associated with) the builds for each component. Everything is mushed together.

**Comment by rljacob** [comment]
I don't necessarily want them separated out.  I just want case.build to report what is happening and point to the location of e3sm.log and not go quiet when cmake kicks in.

**Comment by sarats** [comment]
I do think there is value in getting separate logs to debug issues etc. Just wondering if there would be a way to separate out logs as components will be different CMake targets?
If possible, compilation timing info is also useful to figure out if certain optimizations for files should be turned off etc. 
Just my $0.02.

**Comment by rljacob** [comment]
I'd like to see a message after csm_share like:
Building rest of model with output to file /compyfs/jaco316/e3sm_scratch/Ftest/bld/e3sm.bldlog.200804-185823

**Comment by jonbob** [comment]
Maybe there could be an option to separate output by component. It certainly is one of the things I find most irritating about the current cmake system -- it can be incredibly painful to figure out where errors are coming from.

**Comment by jgfouca** [comment]
Fixed by #3918
